src/tasks/cache/bundle.rs

```diff
@@ -89,10 +89,12 @@ impl BundlePoller {
                 .await
                 .inspect_err(|err| debug!(%err, "Error fetching bundles"))
             {
+                let _guard = span.entered();
                 debug!(count = ?bundles.len(), "found bundles");
                 for bundle in bundles.into_iter() {
```

Comment:
could we add a trace or debug log for the bundle id of each bundle?

Reply:
that should be done in the cache on ingestion, rather than here